src/Administration/Resources/app/administration/src/app/component/data-grid/sw-data-grid/index.js

```diff
@@ -198,10 +198,23 @@ Component.register('sw-data-grid', {
             };
         },
 
+        // @deprecated tag:v6.4.0
         localStorageItemKey() {
             return `${this.identifier}-grid-columns`;
         },
 
+        localStorageGridColumnsItemKey() {
```

Comment:
Wouldnt it be better to use one local storage entry for the whole grid configuration?

Reply:
Good suggestion! I updated the code to use only one storage entry for the user settings.

Reply:
@taltholtmann I go a CI failure and pushed a fix. CI is green now.